draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -373,6 +373,25 @@ The parameter fixedInfo MAY contain any of the following information:
 
 This is a non-comprehensive list, further information can be found in paragraph 5.8.2 of NIST SP800-56Ar3 {{SP800-56A}}.
 
+## Note on the order of parameters
+
+For a two-KEM instantiation, the construction is
+
+```
+KDF(counter || ct_1 || rlen(ct_i1 || ss_1 || rlen(ss_1) || 
+               ct_2 || rlen(ct_2) || ss_2 || rlen(ss_2) || 
+               k_n || fixedInfo, outputBits)
```

Comment:
```suggestion
KDF(counter || ct_1 || rlen(ct_1) || ss_1 || rlen(ss_1) || 
               ct_2 || rlen(ct_2) || ss_2 || rlen(ss_2) || 
               fixedInfo, outputBits)
```

Reply:
Fixed.

Reply:
k_n is still there :sweat_smile:

Reply:
Oh, I missed that there were two typos in there. Got it.